pkg/kubelet/cm/dra/manager.go

```diff
@@ -126,27 +113,11 @@ func (m *ManagerImpl) PrepareResources(pod *v1.Pod) error {
 
 				klog.V(3).InfoS("NodePrepareResource succeeded", "response", response)
 
-				// NOTE: Passing CDI device names as annotations is a temporary solution
-				// It will be removed after all runtimes are updated
-				// to get CDI device names from the ContainerConfig.CDIDevices field
-				annotations, err := generateCDIAnnotations(resourceClaim.UID, driverName, response.CdiDevices)
+				claimInfo, err := newClaimInfo(driverName, resourceClaim.UID, resourceClaim.Name, resourceClaim.Namespace, string(pod.UID), response.CdiDevices)
 				if err != nil {
-					return fmt.Errorf("failed to generate container annotations, err: %+v", err)
+					return err
```

Comment:
Please wrap this error similar to how the other errors in this function are wrapped.